brands/ghostery/branding/pref/Better-Fox.js

```diff
@@ -1,2 +1,291 @@
+
+/** MOZILLA ***/
+// pref("geo.provider.network.url", "https://location.services.mozilla.com/v1/geolocate?key=%MOZILLA_API_KEY%");
+pref("geo.provider.network.logging.enabled", false);
+pref("extensions.blocklist.enabled", true);
+pref("extensions.webextensions.tabhide.enabled", false);
+
+/** TELEMETRY ***/
+// pref("datareporting.policy.dataSubmissionEnabled", false);
+// pref("datareporting.healthreport.uploadEnabled", false);
+// pref("browser.ping-centre.telemetry", false);
+// pref("default-browser-agent.enabled", false);
+// pref("app.shield.optoutstudies.enabled", false);
+// pref("app.normandy.enabled", false);
+// pref("app.normandy.api_url", "");
+// pref("toolkit.telemetry.unified", false);
+// pref("toolkit.telemetry.enabled", false);
+// pref("toolkit.telemetry.archive.enabled", false);
+// pref("toolkit.telemetry.newProfilePing.enabled", false);
+// pref("toolkit.telemetry.shutdownPingSender.enabled", false);
+// pref("toolkit.telemetry.updatePing.enabled", false);
+// pref("toolkit.telemetry.bhrPing.enabled", false);
+// pref("toolkit.telemetry.firstShutdownPing.enabled", false);
+// pref("toolkit.telemetry.coverage.opt-out", true);
+// pref("toolkit.coverage.endpoint.base", "");
+// pref("browser.discovery.enabled", false);
+// pref("breakpad.reportURL", "");
+// pref("browser.tabs.crashReporting.sendReport", false);
+// pref("browser.crashReports.unsubmittedCheck.enabled", false);
+// pref("browser.crashReports.unsubmittedCheck.autoSubmit2", false);
+// pref("browser.newtabpage.activity-stream.feeds.telemetry", false);
+// pref("browser.newtabpage.activity-stream.telemetry", false);
+// pref("extensions.abuseReport.enabled", false);
+// pref("corroborator.enabled", false)
+
+/****************************************************************************
+ * SECTION: PESKYFOX                                                        *
+****************************************************************************/
+
+/** MOZILLA UI ***/
+// pref("toolkit.legacyUserProfileCustomizations.stylesheets", true);
+// pref("browser.privatebrowsing.vpnpromourl", "");
+// pref("extensions.getAddons.showPane", false);
+// pref("extensions.htmlaboutaddons.recommendations.enabled", false);
+// pref("browser.shell.checkDefaultBrowser", false);
+// pref("identity.fxaccounts.enabled", false);
+// pref("browser.aboutwelcome.enabled", false);
+// pref("browser.newtabpage.activity-stream.asrouter.userprefs.cfr.addons", false);
+// pref("browser.newtabpage.activity-stream.asrouter.userprefs.cfr.features", false);
+// pref("security.dialog_enable_delay", 0);
+// pref("extensions.webextensions.restrictedDomains", "accounts-static.cdn.mozilla.net,accounts.firefox.com,addons.cdn.mozilla.net,api.accounts.firefox.com,content.cdn.mozilla.net,discovery.addons.mozilla.org,install.mozilla.org,oauth.accounts.firefox.com,profile.accounts.firefox.com,support.mozilla.org,sync.services.mozilla.com");
```

Comment:
This pref was decided before I joined Ghostery at the community level. You may want to open an issue concerning this to get an official answer on annoyance vs. security benefit.

(This pref is commented out and slotted for removal here because it is included in the `firefox-branding.js`.)